include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -309,7 +333,6 @@ MGConstrainedDoFs::initialize(const DoFHandler<dim, spacedim> &dof)
 }
 
 
-
```

Comment:
Can you add back this line?

Reply:
Thanks for the review. I added the line again in the last commit.